src/ui/popup/manage-settings-page/sync-config.tsx

```diff
@@ -0,0 +1,19 @@
+import {m} from 'malevic';
+import {ViewProps} from '../types';
+import CheckButton from '../check-button';
+
+export default function SyncConfigButton(props: ViewProps) {
+    function syncConfig() {
+        if (!props.data.settings.syncSitesFixes) {
```

Comment:
Remove `if`. Use argument `syncSitesFixes: boolean` for this event handler. It should change the setting in any case. Config should be loaded like `props.actions.changeSettings({syncSitesFixes}); props.actions.loadConfig({local: !syncSitesFixes});`